analyzers/tests/SonarAnalyzer.Test/AnalysisContext/SonarAnalysisContextBaseTest.ShouldAnalyzeTree.cs

```diff
@@ -66,7 +66,7 @@ public void ShouldAnalyzeTree_Scanner_UnchangedFiles_ContainsTreeFile()
     [TestMethod]
     public void ShouldAnalyzeTree_Scanner_UnchangedFiles_ContainsOtherFile()
     {
-        var options = CreateOptions(new[] { "ThisIsNotInCompilation.cs", "SomethingElse.cs" });
+        var options = CreateOptions(["ThisIsNotInCompilation.cs", "SomethingElse.cs"]);
```

Comment:
Out of curiosity: What made you touch this file?

Reply:
excessive zeal I think.
I'm reverting.